### PR TITLE
fixed children alignment issue

### PR DESCRIPTION
**Pull request by shreekeshmurkar**
Fixed following issue of children content alignment:
https://github.com/Hacker0x01/react-datepicker/issues/3654

Before fix:
image

After fix:
image

### CODE REVIEW

**Comment by shreekeshmurkar** [comment]
@martijnrusschen 
Please review

**Comment by shreekeshmurkar** [comment]
Please review

File: src/calendar.jsx [hunk 1]
```diff
@@ -978,6 +978,16 @@ export default class Calendar extends React.Component {
     }
   };
 
+  renderChildren = () => {
```
Comment (martijnrusschen) [inline src/calendar.jsx:981]: please add a test to cover this

Comment (shreekeshmurkar) [reply src/calendar.jsx:981]: Added tests

**Comment by shreekeshmurkar** [comment]
@martijnrusschen
Please review

**Pull request merged**